tests/unit/s2n_fork_generation_number_test.c

```diff
@@ -190,7 +190,7 @@ static int s2n_unit_test_clone_child_process(void *parent_process_fgn)
     EXPECT_EQUAL(s2n_unit_test_thread(return_fork_generation_number), S2N_SUCCESS);
 
     /* This translates to the exit code for this child process */
```

Comment:
nit - I think this comment isn't needed anymore, since we're exiting directly and not propagating the exit value through `clone`.
```suggestion
```